include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -254,12 +307,26 @@ __pattern_find_if(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _Range&&
 }
 
 #if _ONEDPL_CPP20_RANGES_PRESENT
+template <typename _Pred, typename _Proj>
+struct __pattern_find_if_pred
```

Comment:
It seems we already have the same (with functionality perspective) type above, on line 96

struct __pattern_for_each_fn.

Reply:
Fixed